app/styles/button.module.css

```diff
@@ -53,3 +53,49 @@
   transition: all 0.5s ease;
   width: 14rem;
 }
+
+.action-buttons {
+  margin-bottom: 20px;
```

Comment:
styling in px doesn't seems right to me. It is recommended to use rem instead. 
Ref : https://www.freecodecamp.org/news/why-use-rem-to-set-font-size-in-css/#:~:text=For%20styling%20a%20webpage%2C%20we,the%20size%20of%20the%20screen.